Cpp/odin-views/password.cpp

```diff
@@ -185,5 +186,45 @@ namespace {
         }
     } c_reset_password;
 
+    const class password_hash : public fostlib::urlhandler::view {
+      public:
+        password_hash() : view("odin.password.hash") {}
+
+        std::pair<boost::shared_ptr<fostlib::mime>, int> operator()(
+                const fostlib::json &config,
+                const fostlib::string &path,
+                fostlib::http::server::request &req,
+                const fostlib::host &host) const {
+            if (!config.has_key("hash") || !config.has_key("verify")
+                || !config.has_key("then")) {
+                throw fostlib::exceptions::not_implemented(
+                        __PRETTY_FUNCTION__,
+                        "Must supply both hash, verify and then in the "
```

Comment:
I couldn't parse that text to start with. Maybe put single quotes around the key words?

    "Must supply 'hash', 'verify' and 'then' in the "

Reply:
Although `verify` is presumably optional, but highly recommended?

Reply:
for now, I think it's ok if you set the config to be the same as `hash` key.
e.g.```
{"hash": ["body", "password"], "verify": ["body", "password"]}
```